packages/open_aea/connections/p2p_libp2p/libp2p_node/utils/utils_test.go

```diff
@@ -421,3 +431,46 @@ func TestBootstrapConnect(t *testing.T) {
 	})
 
 }
+
+// Ethereum tests
+// address: 0xfF516F1273232fB64936FF03DA6eC85108b1d09A
+// private: 0xbb0c01836c9ddfc89a890d829dfaa569be545bac71cf20bbff8e02a114a2f042
+// public: 0x4a47e8a74fab63f0a8e7615cc9776960159bc79cefc9b6e3164c4c4e018247f58ee51a200a4286fb49af6246c1e14649395a5e658209dbc6086c89530acf7ade
+
+func TestHexutilDecodeEncode(t *testing.T) {
+	keyHex := "0xbb0c01836c9ddfc89a890d829dfaa569be545bac71cf20bbff8e02a114a2f042"
+	keyBytes, err := hexutil.Decode(keyHex)
+	assert.Equal(t, nil, err)
+	assert.Equal(t, hexutil.Encode(keyBytes), keyHex)
+}
+
+func TestPublicKeyAndAddressFromPrivateKey(t *testing.T) {
```

Comment:
You are not testing any of the code you write, you are just testing external libs. Why?

You need to test your methods KeyPairFromEthereumKey and EthereumPublicKeyFromPubKey and IdentityFromEthereumKey

Reply:
I know. And I like to think that if it had not taken me so long to look up the information I needed online that I would have implemented it as part of my functions already.